lib/mayaUsd/utils/util.h

```diff
@@ -180,6 +181,10 @@ MString GetUniqueNameOfDagNode(const MObject& node);
 MAYAUSD_CORE_PUBLIC
 MStatus GetMObjectByName(const std::string& nodeName, MObject& mObj);
 
+/// Gets the UsdStage for the proxy shape  node named \p nodeName.
```

Comment:
I would add a comment pointer here to lib/mayaUsd/ufe/Utils.h

Reply:
Sorry, you just want me to mention in the comments that there is a similar function that uses UFE over in a different utilities file?  
I believe the UFE approach would be to:
Convert the MDagPath into a UFE PathSegment.
Convert the PathSegment into a UFE Path.
Use the stage map to go from a Path to a stage ptr.

The only issue with the Maya way is what the performance is when you have a unique dag path and want the depend node, which I don't think is all that bad.  Once you have the node then it's just a cast to our base class and we call the function on it to return the stage ptr.

Reply:
That seems very roundabout...  If you have an MDagPath, you don't need to go through UFE!  You can just get the MObject from the tail of the path, and get a pointer from that.

And yes, just add a comment saying if you already have a dependence on UFE, then lib/mayaUsd/ufe/Utils.h is better.